src/views/manage-patches-view/layout.tsx

```diff
@@ -20,7 +20,7 @@ export const ManagePatchesLayout = ({ assetId }: { assetId: string | undefined }
         </Link>
       </BrowserRouter>
       {showSuccess && (
-        <StatusBox variant="success" title="The patches have been updated successfully">
+        <StatusBox variant="success" title="The patch has been updated successfully">
```

Comment:
This message should say something like:
"The update has completed successfully"
Only because you can edit a patch or area record so it should be suitable for both scenarios hence a generic message should be fine IMO

Reply:
Set the message to "The update has completed successfully" in 33b5daecbec7a4fc9e89d7ace12c70fab9fa2d30